deploy/helm-preview/services/headless-example/templates/configmap.yaml

```
@@ -0,0 +1,17 @@
+{{- $name := .Release.Name }}
+{{- $namespace:= .Release.Namespace }}
+{{- if .Values.headlessexample.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: {{ .Values.headlessexample.nameOverride }}
+    namespace: {{ .Release.Namespace | quote }}
+    labels:
+      app: {{ .Values.headlessexample.nameOverride }}
+data:
+  {{- range $key, $value := .Values.headlessexample.applicationConfig }}
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)